core/tests/protractor/learnerDashboard.js

```diff
@@ -41,210 +41,218 @@ describe('Learner dashboard functionality', function() {
+    subscriptionDashboardPage =
+      new SubscriptionDashboardPage.SubscriptionDashboardPage();
   });
 
-  beforeAll(function() {
-    adminPage = new AdminPage.AdminPage();
-    collectionEditorPage = new CollectionEditorPage.CollectionEditorPage();
-    learnerDashboardPage = new LearnerDashboardPage.LearnerDashboardPage();
-    // Create a new learner.
+  it('displays completed explorations', function() {
     users.createUser('learner@learnerDashboard.com', 'learnerlearnerDashboard');
-    users.createUser(
-      'creator2@learnerDashboard.com', 'creator2learnerDashboard');
     users.createModerator(
-      'creator3@learnerDashboard.com', 'creator3learnerDashboard');
-
-    var USERNAME = 'creator1learnerDashboard';
-    users.createAndLoginAdminUser('creator1@learnerDashboard.com', USERNAME);
-    adminPage.reloadAllExplorations();
-    adminPage.updateRole(USERNAME, 'collection editor');
-    browser.get(general.SERVER_URL_PREFIX);
-    var dropdown = element(by.css('.protractor-test-profile-dropdown'));
-    browser.actions().mouseMove(dropdown).perform();
-    dropdown.element(by.css('.protractor-test-dashboard-link')).click();
-    browser.waitForAngular();
-    element(by.css('.protractor-test-create-activity')).click();
-    // Create new collection.
-    element(by.css('.protractor-test-create-collection')).click();
-    browser.waitForAngular();
-    collectionEditorPage.addExistingExploration('14');
-    collectionEditorPage.saveDraft();
-    collectionEditorPage.closeSaveModal();
-    collectionEditorPage.publishCollection();
-    collectionEditorPage.setTitle('Test Collection');
-    collectionEditorPage.setObjective('This is a test collection.');
-    collectionEditorPage.setCategory('Algebra');
-    collectionEditorPage.saveChanges();
-    browser.waitForAngular();
-    users.logout();
-  });
+      'moderator@learnerDashboard.com', 'moderatorlearnerDashboard');
+    users.createAndLoginAdminUser(
+      'admin@learnerDashboard.com', 'collectionAdmlearnerDashboard');
 
-  it('displays incomplete and completed explorations', function() {
-    users.login('learner@learnerDashboard.com');
+    adminPage.get();
+    // Load exploration '3'
+    adminPage.reloadExploration('root_linear_coefficient_theorem.yaml');
+    // Load exploration '14'
+    adminPage.reloadExploration('about_oppia.yaml');
 
-    // Play an exploration and leave it in between. It should be added to the
-    // 'In Progress' section.
-    general.openPlayer('3');
-    explorationPlayerPage.submitAnswer('Continue', null);
-    browser.ignoreSynchronization = true;
-    learnerDashboardPage.get();
-    general.acceptAlert();
-    browser.ignoreSynchronization = false;
-    browser.waitForAngular();
-    libraryPage.expectExplorationToBeVisible('Root Linear Coefficient Theorem');
+    users.login('learner@learnerDashboard.com');
+    // TODO(hoangviet1993): Play an exploration and leave it in between and
```

Comment:
I am current investigating as  browser.ignoreSynchronization is currently deprecated and replaced by browser.waitForAngularEnabled. 

Also the other issue was this tests want to load exploration from the .yaml file which is apparently a problem for Travis. I went ahead and made the test create its own exploration for each test suite now :+1:

Reply:
That is awesome. Thanks!